inspirehep/modules/converttohal/tei.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+
+from jinja2 import Environment, PackageLoader
+import json
+
+TEMPLATE = 'default.xml'
+#from inspirehep.utils.record_getter import get_es_record
+from inspirehep.modules.records.json_ref_loader import replace_refs
+#from inspirehep.modules.authors.utils import scan_author_string_for_phrases
+from nameparser import HumanName
+import sys
+
+
+def tei_response(record):
+    
```

Comment:
Top-level functions should have a docstring, a particular comment which becomes the `__doc__` attribute of the function and acts as an inline help when using an interactive environment (like IPython). Docstrings formatting has its own PEP: https://www.python.org/dev/peps/pep-0257/